lib/blog-feed-loader.js

```diff
@@ -1,2 +1,6 @@
-// TODO: Replace this with a real implementation.
-module.exports = require('../test/browser/stub-blog-feed-loader');
+if (process.env.NODE_ENV !== 'production' &&
```

Comment:
FWIW we're also already doing this kind of thing with the [dev ribbon](https://github.com/mozilla/teach.webmaker.org/blob/develop/components/page.jsx#L9-L12):

```js
var DevRibbon = (process.env.NODE_ENV === 'production' &&
                 process.env.SHOW_DEV_RIBBON !== 'on')
                ? null
                : require('./dev-ribbon.jsx');
```

because of the way we're writing it, we'll ensure that `dev-ribbon.jsx` isn't included in production builds (well, production builds that don't also have `SHOW_DEV_RIBBON` on).